sdk/samples/iot/sample_pnp_device_info_component.h

```diff
@@ -4,20 +4,26 @@
 #ifndef SAMPLE_PNP_DEVICE_INFO_COMPONENT_H
 #define SAMPLE_PNP_DEVICE_INFO_COMPONENT_H
 
+#include "sample_pnp_mqtt_component.h"
+
 #include <azure/core/az_result.h>
-#include <azure/core/az_span.h>
 #include <azure/iot/az_iot_hub_client.h>
 
+typedef struct
+{
+  az_span component_name;
+} pnp_device_info_component;
```

Comment:
is this used anywhere?

Reply:
no.  i'm not sure why I put it in either...